packages/integration-tests/tests/api-resources.test.ts

```diff
@@ -0,0 +1,77 @@
+import { Resource } from '@logto/schemas';
+import { managementResource } from '@logto/schemas/lib/seeds';
+
+import { authedAdminApi } from '@/api';
+import { generateResourceIndicator, generateResourceName } from '@/utils';
+
+const createResource = (name?: string, indicator?: string) =>
+  authedAdminApi
+    .post('resources', {
+      json: {
+        name: name ?? generateResourceName(),
+        indicator: indicator ?? generateResourceIndicator(),
```

Comment:
Do we have to generate a random resourceName & Indicator?  Or just manually gibe some static value is fine?

Reply:
The indicator is unique, and I also want to keep creating resources with different names.